test/common/cdp.py

```diff
@@ -196,8 +196,13 @@ def invoke(self, fn, **kwargs):
         if fn == 'Runtime.evaluate':
             cmd = "%s, contextId: getFrameExecId(%s)%s" % (cmd[:-2], jsquote(self.cur_frame), cmd[-2:])
 
+        waitPageLoad = fn in ['Page.navigate', 'Page.reload']
```

Comment:
I am not too excited about this bit, tbh. What about using a kwarg to activate this behavior explicitly instead of doing it implicitly via the function name.

Reply:
I thought about it, but then didn't like it. This is intrinsic behaviour of these two functions, not something that the caller can even decide on. It would be sort of an "unbreak my funtion call" option, and it's easy to forget. I.e. I'd rather list them in a central location.

But I'm happy to discuss it further of course!

Reply:
Sounds good!